src/main/java/com/hobbycollection/HobbyCollectionApplication.java

```diff
@@ -7,6 +7,7 @@
 public class HobbyCollectionApplication {
 
     public static void main(String[] args) {
+
```

Comment:
I fixed code formatting / indentation.
As best practice is to use four spaces, a unit of indentation.
Source: https://www.baeldung.com/java-clean-code